tests/promptflow/flow_with_additional_includes/hello.py

```diff
@@ -0,0 +1,64 @@
+import os
+import openai
+
+from dotenv import load_dotenv
+from promptflow import tool
+
+# The inputs section will change based on the arguments of the tool function, after you save the code
+# Adding type to arguments and return value will help the system show the types properly
+# Please update the function name/signature per need
+
+
+def to_bool(value) -> bool:
+    return str(value).lower() == "true"
+
+
+@tool
+def my_python_tool(
+    prompt: str,
+    deployment_name: str,
+    suffix: str = None,
+    max_tokens: int = 120,
+    temperature: float = 1.0,
+    top_p: float = 1.0,
+    n: int = 1,
+    logprobs: int = None,
+    echo: bool = False,
+    stop: list = None,
+    presence_penalty: float = 0,
+    frequency_penalty: float = 0,
+    best_of: int = 1,
+    logit_bias: dict = {},
```

Comment:
```suggestion
    logit_bias: dict = None,
```

We should not use a mutable object as a default value.